mivs/models.py

```diff
@@ -0,0 +1,206 @@
+from mivs import *
+
+
+@Session.model_mixin
+class SessionMixin:
+    def logged_in_studio(self):
+        try:
+            return self.indie_studio(cherrypy.session['studio_id'])
+        except:
+            raise HTTPRedirect('../mivs_applications/start')
+
+    def delete_screenshot(self, screenshot):
+        self.delete(screenshot)
+        try:
+            os.remove(screenshot.filepath)
+        except:
+            pass
+        self.commit()
+
+
+@Session.model_mixin
+class AdminAccount:
+    judge = relationship('IndieJudge', uselist=False, backref='admin_account')
+
+
```

Comment:
A number of the models in this file are not yet used.  We're currently using ``IndieStudio``, ``IndieGame``, ``IndieDeveloper``, ``IndieGameCode``, and ``IndieGameScreenshot`` and the rest are here because they will come into play once the admin side is implemented.